CDCgov/prime-reportstream
# 14209 - Fixed Daily data render bug.

This PR ...

Fixes a rendering bug that appears on the the Daily Data filter section.

Test Steps:
1. Log into RS
2. Go to Daily Data
3. Check that the filter display looks correct.

*I was able to repo locally by:
1. Selecting `ak-phd` 
2. Select Submissions (locally I have no data being returned)
3. Select Daily Data

## Changes
- css changes

## Checklist

### Testing

## Linked Issues
- Fixes #14209
